src/containers/Versions/GroupedNodesTree/GroupedNodesTree.tsx

```diff
@@ -15,83 +14,80 @@ const b = cn('ydb-versions-grouped-node-tree');
 
 interface GroupedNodesTreeProps {
     title?: string;
+    isDatabase?: boolean;
     nodes?: NodesPreparedEntity[];
     items?: GroupedNodesItem[];
     expanded?: boolean;
     versionColor?: string;
-    versionsValues?: VersionValue[];
-    level?: number;
+    preparedVersions?: PreparedVersion[];
 }
 
 export const GroupedNodesTree = ({
     title,
+    isDatabase,
     nodes,
     items,
     expanded = false,
     versionColor,
-    versionsValues,
-    level = 0,
+    preparedVersions,
 }: GroupedNodesTreeProps) => {
-    const [isOpened, toggleBlock] = React.useState(false);
+    const [isOpened, setIsOpened] = React.useState(false);
 
     React.useEffect(() => {
-        toggleBlock(expanded);
+        setIsOpened(expanded);
```

Comment:
whats the difference between isOpened and expanded here?

Maybe better to pass setExpanded from props and remove local state and useEffect